recipes/stlab/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, tools
+import os
+
+class Stlab(ConanFile):
+    name = 'stlab'
+    description = 'The Software Technology Lab libraries.'
+    url = 'https://github.com/stlab/libraries'
+    homepage = 'https://github.com/stlab/libraries'
+    author = 'Manu Sanchez and Tim van Deurzen'
+    license = 'Boost Software License 1.0'
```

Comment:
```suggestion
    license = 'BSL-1.0'
```
Use short identifier: see https://spdx.org/licenses/BSL-1.0.html